content/log/index.md

```diff
@@ -0,0 +1,141 @@
+
+If you don't find a log section in your file you need to use the custom file configuration below instead. 
+
+<div class="alert alert-warning">
+You might not find any log section in your yaml files during the private beta, however please find below the list of integration and their yaml file that are supported:
+</div>
+
+* Apache: [apache.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/apache/conf.yaml.example)
+* Haproxy: [haproxy.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/haproxy/conf.yaml.example)
+* IIS: [iis.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/iis/conf.yaml.example)
+* Java: jmx.yaml.example
+* Mongodb: [mongodb.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/mongo/conf.yaml.example)
+* Nginx: [nginx.yaml.example](https://github.com/DataDog/integrations-core/blob/nils/Logs-integration-beta/nginx/conf.yaml.example)
+
+## Custom log collection
+
+To send custom log file that would not be part of an existing integration. Rename `custom-logs.yaml.example` in `custom-logs.yaml` and edit those parameter:
+
+
+* `type` : (manatory) type of log input source (**tcp** / **udp** / **file**)
+* `port` / `path` : (mandatory) option depending on `type`.
+* `service` : (mandatory) name of the service owning the log
+* `source` : (mandatory) attribute that defines which integration is sending the logs. It can be a custom one if the integration does not exist
+* `sourcecategory` : (optional) Multiple value attribute. Can be used to refine the source attribtue. Example: source:mongodb, sourcecategory:db_slow_logs
+* `tags`: (optional) add tags to each logs collected.
+
+### Tail existing files
+Set `type` to **file** then specify the absolute `path` to the log file you want to tail.
+
+Example: 
+If you want to gather you python app logs for instance stored in **/var/log/myapp1.log** and **/var/log/python.log** you would edit your `custom-logs.yaml` file as follow
+
+{{< highlight yaml >}}
+init_config:
+instances:
+  whatever: anything
+#Log section
+logs:
+
+  type: file
+  path: /var/log/myapp1.log
+  service: myapp1
+  source: python
+  sourcecategory: sourcecode
+  tags: env:prod
+
+  type: file
+  path: /var/log/python.log
+  service: myapplication
+  source: python
+  sourcecategory: sourcecode
+{{< /highlight >}}
+
+### Stream logs through TCP/UDP
+Set `type` to **tcp** or **udp** depending of your protocol then specify the `port` of your incomming connection.
+
+Example: 
+If your php application doesn't log into a file but forwards its logs on the local 10518 port in TCP edit your `custom-logs.yaml` file as such in order to listen to this port and forward those logs to your datadog application.:
```

Comment:
s/php/PHP

> but forwards its logs on the local 10518 port in TCP

but forwards its logs locally to TCP port 10518

Reply:
👍